tools/serverpod_cli/lib/src/generator/open_api/open_api_definition.dart

```diff
@@ -0,0 +1,200 @@
+library openapi_definition;
+
+import 'package:recase/recase.dart';
+
+import '../../../analyzer.dart';
+import '../../analyzer/dart/definitions.dart';
+import '../../logger/logger.dart';
+
+part 'open_api_objects.dart';
+part 'open_api_utils.dart';
+part 'open_api_extensions.dart';
+
+// ignore_for_file: public_member_api_docs, sort_constructors_first
+/// OpenAPI Object
+/// This is the root object of the OpenAPI document.
+class OpenApiDefinition {
+  final String openApi;
+
+  /// REQUIRED. Provides metadata about the API.
```

Comment:
No need to include the `REQUIRED` part, as it will be implicit by the constructor.

Reply:
Also, avoid capital letters in comments (`MAY`)